### DISCUSSION TITLE
DAAN复现

### DISCUSSION THREAD

**Discussion by hanjingyu101** [jindongwang/transferlearning#347] [Q&A] [answered]
老师您好，在运行.sh的过程中出现yaml文件找不到相应路径，改怎么解决呢？
![迁移学习 sh文件复现](https://user-images.githubusercontent.com/111672273/185787074-40220a39-df47-4444-8b26-e87b2e74f179.png)

**Comment by jindongwang** [comment, marked as answer]
这是基础问题，你写对路径就可以了。
